molecule/backup/library/check_backup_files_list.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+import os
+import re
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+argument_spec = {
+    'instance_info': {'required': True, 'type': 'dict'},
+    'files_list': {'required': True, 'type': 'list'},
+    'stateboard': {'required': True, 'type': 'bool'},
+}
+
+
+if __name__ == '__main__':
+    module = AnsibleModule(argument_spec=argument_spec)
+    params = module.params
+
+    files_list = params['files_list']
+    instance_info = params['instance_info']
+    stateboard = params['stateboard']
+
+    files_list = [
+        path.strip("/") for path in files_list
+    ]
```

Comment:
```suggestion
    files_list = map(lambda path: path.strip(os.path.sep), files_list)
```

Cast to list is not necessary, because it will be used in filter function.

Also, it can be moved down to filter.